examples/02_decoding/plot_haxby_searchlight.py

```diff
@@ -23,7 +22,6 @@
 print('Functional nifti image (4D) is located at: %s' % haxby_dataset.func[0])
 
 fmri_filename = haxby_dataset.func[0]
-fmri_img = nibabel.load(fmri_filename)
```

Comment:
I don't think that the example runs, now: you need to change 'fmri_img' to 'fmri_filename' below (on line 33).